TLM/TLM/UI/LegacySubTool.cs

```diff
@@ -185,6 +184,11 @@ public abstract class LegacySubTool : IObserver<ModUI.EventPublishers.UIOpacityN
         public virtual void ShowGUIOverlay(ToolMode toolMode, bool viewOnly) { }
 
         public virtual bool IsCursorInPanel() {
+            return IsCursorInPanel_();
+        }
+
+        /// <summary>Check whether cursor is in the main menu or debug menu area.</summary>
+        internal static bool IsCursorInPanel_() {
```

Comment:
Remind me, why we use that weird style? 
I've never seen it in any C# code I worked with before, it's against method naming from MS C# rules etc.
I think we should install some automated linter or fix ruleset to be as close as possible (without changing too much) to the standard MS layout/naming C# guidelines.

Reply:
I see it fails because we allow underscore in method names when they are long so maybe it's not as strict as it could be? 🤷 There is .editorconfig in the project

Anyways, according to standard MS guidelines, underscore in **method names** _(prefix/suffix)_ (there's one for debug logging but that would create too much noise if we try to change it), **variable names**, **method params** are not allowed.

There are some PR's in progress or with planned features so we should try to keep those rules in mind just for new code for now (or if big part of some file is changed)